Source/RunActivity/Viewer3D/Map/MapDataProvider.cs

```diff
@@ -253,40 +253,42 @@ public float GetUnusedYLocation(float startX, float wantY, string name)
                 return noFreeSlotFound;
 
             var positionY = desiredPositionY;
-            while (positionY >= 0 && positionY < F.alignedTextY.Length)
-            {
-                // If the line contains no text yet, put it there
-                if (F.alignedTextNum[positionY] == 0)
-                    return SaveLabelLocation(startX, endX, positionY);
+            if (F.alignedTextY != null) {
```

Comment:
This extra if had to be added, otherwise during startup this code failed on F.alignedTextY being null. Tried to do the setting of the defaults at other place during start of the Form, but no luck.